server/src/service-docs/service-docs.service.spec.ts

```diff
@@ -102,3 +109,62 @@ describe('ServiceDocsService', () => {
     expect(found).toEqual(expectedServiceDoc);
   });
 });
+
+describe('ServiceDocs ORM conversion', () => {
+  it('should convert valid extension fields', () => {
+    const model = {
+      name: 'test',
+      extensions: {
+        'x-test1': 123,
+        'x-test2': 'asd',
+        'x-test3': false,
+        'x-test4': [123, 'asd', false],
```

Comment:
I think we don't need the "x" prefix anymore. Should we maybe remove this prefix from the tests as well?

The same goes for the e2e test.